e2e/tenant_control_plane_ready_test.go

```diff
@@ -0,0 +1,88 @@
+package e2e
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"k8s.io/apimachinery/pkg/types"
+
+	kamajiv1alpha1 "github.com/clastix/kamaji/api/v1alpha1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var _ = Describe("Deploy a TenantControlPlane resource", func() {
+	// Fill TenantControlPlane object
+	tcp := kamajiv1alpha1.TenantControlPlane{
+		TypeMeta:   metav1.TypeMeta{},
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "tenant-test",
+			Namespace: "kamaji-system",
+		},
+		Spec: kamajiv1alpha1.TenantControlPlaneSpec{
+			ControlPlane: kamajiv1alpha1.ControlPlane{
+				Deployment: kamajiv1alpha1.DeploymentSpec{
+					Replicas: 2,
+				},
+				Ingress: kamajiv1alpha1.IngressSpec{
+					Enabled: true,
+				},
+				Service: kamajiv1alpha1.ServiceSpec{
+					ServiceType: "NodePort",
+				},
+			},
+			NetworkProfile: kamajiv1alpha1.NetworkProfileSpec{
+				Address: "172.18.0.2",
+				DNSServiceIPs: []string{"10.96.0.10"},
+				Domain: "clastix.labs",
+				PodCIDR: "10.244.0.0/16",
+				Port: 31443,
+				ServiceCIDR: "10.96.0.0/16",
+			},
+			Kubernetes: kamajiv1alpha1.KubernetesSpec{
+				Version: "v1.23.6",
+				Kubelet: kamajiv1alpha1.KubeletSpec{
+					CGroupFS: "cgroupfs",
+				},
+				AdmissionControllers: kamajiv1alpha1.AdmissionControllers{
+					"LimitRanger",
+					"ResourceQuota",
+				},
+			},
+		},
+		Status:     kamajiv1alpha1.TenantControlPlaneStatus{},
```

Comment:
JFI, there's no need of the status here: it's a subresource, so it cannot be created, just updated.

Reply:
So, I'm going to remove it